### DISCUSSION TITLE
Add more object-types to ChangeLog filter

### DISCUSSION THREAD

**Discussion by PieterL75** [netbox-community/netbox#8238] [Q&A]
### NetBox version

v3.1.3

### Feature type

Change to existing functionality

### Proposed functionality

The ChangeLog filter currently has a limited set of object types.
I'm missing the Prefix and IP Address for example.

Can the object filter be extended with all objects that could log anything ?

ty !

Pieter

### Use case

Looking for a changed IP address in the Changelog is pretty hard now.

### Database changes

none

**Comment by jeremystretch** [comment]
I see all the expected object types on v3.1.3 when filtering the change log. Where specifically are you looking?

**Comment by PieterL75** [comment]
I don't have any ipam related objects.
Running v3.1.3 also.
I'll post screenshot tomorrow

**Comment by DanSheps** [comment]
They appear all present:

![Screenshot_20211231-185836](https://user-images.githubusercontent.com/11049792/147841896-143578d5-053b-4bbd-87cb-1bfa5b09e142.png)

**Comment by PieterL75** [comment]
Its is here : 
https://netbox/extras/changelog/

What I noticed : 
- Don't type anything in the Object Type and scroll down. 
- You will see that IPAM is missing
- Type ipam in the dropdown, and they appear

Seems some pre_fetching that is not 100%

Pieter

**Comment by jeremystretch** [comment]
> Its is here :

We don't have access to your local NetBox installation (nor should we).

This seems to be an issue with your local instance. I suggest re-running the upgrade script to ensure that all static files have been copied, and then forcing a browser refresh (ctrl+F5).

I'm converting this to a discussion as there's no action to be taken by the maintainers.

**Reply by PieterL75** [reply]
I did not intend to post my internal URL here... Just wanted to show the page were I'm having the issue on

The (cosmetic) issue is on 3 different instances of netbox that I'm running, and also on https://demo.netbox.dev/extras/changelog/

Pieter

**Reply by kkthxbye-code** [reply]
The dropdown shows all object types for me on the demo instance and on our internal instances. Sure you are scrolling to the bottom? The dropdown is loaded in three requests (I think it has a max page size of 50 regardless of user setting), auto-filled when reaching the bottom (happens when filtering also).

Could you check chrome dev tools (or equivalent) and check if it tries to load the remaining elements when reaching the bottom? You should be able to see requests towards https://demo.netbox.dev/api/extras/content-types/
